### DISCUSSION TITLE
Fan speed

### DISCUSSION THREAD

**Discussion by Pathfinder1978** [Jyers/Marlin#1463] [Q&A] [answered]
Is it possible to display the fan speed in percent?

**Comment by Cal-LF** [comment, marked as answer]
Others have asked for this as it makes sense to have a percentage like other displayed values. There seems to be a need to hold onto the 0 - 255 value range.

**Reply by Jyers** [reply]
The main benefit is it gives more fine control over the value, but yes it could be tweaked to be a percent. Maybe at some point I'll make it a configurable option
